cmd/bootstrapper/cmd_test.go

```diff
@@ -183,25 +175,22 @@ func TestBootstrapConfigurationStep(t *testing.T) {
 		err := cmd.Execute()
 		require.NoError(t, err)
 
-		exists, err := fs.Exists(filepath.Join(configPath, preload.ConfigPath))
-		require.NoError(t, err)
-		assert.False(t, exists)
+		assert.NoFileExists(t, filepath.Join(configPath, preload.ConfigPath))
 
 		containerConfigPath := filepath.Join(configPath, "test-container")
+		assert.NoFileExists(t, pmc.GetDestinationRuxitAgentProcFilePath(containerConfigPath))
 
-		exists, err = fs.Exists(pmc.GetDestinationRuxitAgentProcFilePath(containerConfigPath))
-		require.NoError(t, err)
-		assert.False(t, exists)
-
-		exists, err = fs.Exists(filepath.Join(containerConfigPath, "enrichment/endpoint/endpoint.properties"))
-		require.NoError(t, err)
-		assert.True(t, exists)
+		assert.FileExists(t, filepath.Join(containerConfigPath, "enrichment/endpoint/endpoint.properties"))
 	})
 
 	t.Run("OA + metadata", func(t *testing.T) {
-		fs := setupFS(t)
+		tmpDir := t.TempDir()
+		targetPath := filepath.Join(tmpDir, "/mnt", "bin")
```

Comment:
may be we can create test helper to have all these prepare steps in one place?
it can be reused 3 times